src/transports/PostMessageIframeTransport.test.ts

```diff
@@ -0,0 +1,85 @@
+import PostMessageTransport from "./PostMessageIframeTransport";
+import { generateMockRequest } from "../__mocks__/requestData";
+import handler from "serve-handler";
+import http from "http";
+
+
+let server: http.Server | undefined;
+
+beforeAll(() => {
+  server = http.createServer((request, response) => {
+    return handler(request, response, {
+      public: "src/testiframe",
+    });
+  });
+  server.listen(5000);
```

Comment:
nit. you should use random 0 ports . So in a ci situation or in a concurrent test situation there are no conflicts.